index.markdown

```diff
@@ -472,6 +472,18 @@ are likely to just disappoint the winning team. When organizations/individuals
 wish to pursue further development of a hack they should speak to the winning
 team separately.
 
+### Attitude to prize winners
+
+Those who have won a prize at a hack day should not be expected to provide any
+ongoing service or favours to sponsors or organisers. Some hack day sponsors
+have been known to ask prize winners to write promotional blog posts for them.
```

Comment:
I'd rather not have a "Some X do" without calling someone out explicitly. Instead we can write this as "Making it mandatory for prize winners to write promotional blog posts is not acceptable".